### DISCUSSION TITLE
Invalid route action

### DISCUSSION THREAD

**Discussion by pravesh-mavens** [spatie/laravel-health#195] [Q&A] [answered]
`use Spatie\Health\Http\Controllers\HealthCheckResultsController;`
`Route::get('health', HealthCheckResultsController::class);`
**Invalid route action: [App\Http\Controllers\Spatie\Health\Http\Controllers\HealthCheckResultsController].**

can you help us why we are facing this issues.

**Comment by pravesh-mavens** [comment]
![image](https://github.com/spatie/laravel-health/assets/57610415/59b94c7f-c2bc-44e6-b1d8-3ba487150639)

**Comment by clementmas** [comment, marked as answer]
Somebody had the same problem (use the search). See https://github.com/spatie/laravel-health/discussions/37
